moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1990,21 +1990,22 @@ double RobotState::computeCartesianPath(const JointModelGroup* group, std::vecto
 }
 
 double RobotState::testJointSpaceJump(const JointModelGroup* group, std::vector<RobotStatePtr>& traj,
-                                      const JumpThreshold& jump_threshold)
+                                      const JumpThreshold& jt)
 {
-  double percentage = 1.0;
-  if (jump_threshold.jump_threshold_factor > 0.0)
-    percentage *= testJointSpaceJump(group, traj, jump_threshold.jump_threshold_factor);
+  if (jt.jump_threshold_factor > 0.0)
+    return testJointSpaceJump(group, traj, jt.jump_threshold_factor);
 
-  if (jump_threshold.prismatic_jump_threshold > 0.0 || jump_threshold.revolute_jump_threshold > 0.0)
-    percentage *= testJointSpaceJump(group, traj, jump_threshold.revolute_jump_threshold,
-                                     jump_threshold.prismatic_jump_threshold);
+  else if (jt.prismatic_jump_threshold > 0.0 && jt.revolute_jump_threshold > 0.0)
```

Comment:
Switched to `else if` here. For me, it doesn't make sense to test for both, relative and absolute jumps.

Reply:
I generally agree